apps/cms/snapshot.yml

```diff
@@ -5750,42 +5807,57 @@ fields:
       options:
         languageTemplate: '{{name}}'
         translationsTemplate: '{{name}}'
-      display: null
-      display_options: null
+        languageField: code
+      display: translations
+      display_options:
+        template: '{{name}}'
+        languageField: code
+        defaultLanguage: en-US
+        userLanguage: true
       readonly: false
       hidden: false
-      sort: null
+      sort: 11
       width: full
       translations: null
       note: null
       conditions: null
       required: false
       group: null
-  - collection: sets
+  - collection: pack_templates
     field: nft_templates
     type: alias
     schema: null
     meta:
-      collection: sets
+      collection: pack_templates
       field: nft_templates
       special:
         - o2m
       interface: list-o2m
       options:
         enableCreate: false
-        enableSelect: true
         template: '{{unique_code}}'
+        filter:
+          _and:
+            - pack_template:
+                id:
+                  _null: true
```

Comment:
Can only select NFT Templates that have not already been associated with a Pack Template.